.github/CONTRIBUTING.md

```diff
@@ -99,58 +99,69 @@ If you got an idea for a component that you think should be part of Kirby you sh
+```
 
 3. Start the cookbook server:
-`npm start`
+```
+npm start
+```
 
-You should be able to access the Cookbook by opening `http://localhost:4200` in a browser.
+You should now be able to open `http://localhost:4200` in a browser to access the cookbook.
 
-#### Node 
+### Node 
 <!-- Which version should people use, and perhaps a recommendation on which tool people can use to manage versions. -->
 [Node](https://nodejs.org/en/) is the runtime used for development. 
+
 We recommend using Node version `INSERT HIGHEST VERSION WE HAVE GOTTEN RUNNING WITHOUT ISSUES`. This is because we have experienced errors with other version due to **What was the problem exactly again?**.  
 
 If you have need other versions of Node installed then a version manager such as [NVM](https://github.com/nvm-sh/nvm) can be used. 
 
-#### Tslint 
+### Tslint 
 <!-- People should make sure to have it enabled to avoid unnecessary comments during code review-->
 **Note:** Work is being done to migrate from TSLint to ESLint (see issue: [#1317](https://github.com/kirbydesign/designsystem/issues/1317)). 
 
 TSLint is used for linting which helps keep up the quality of the submitted code for Kirby. 
 Scripts for running TSLint can be found in `package.json`, but we recommend that you install a TSLint plugin for your editor. It can be useful to enable "fix on save"-functionality such that simple errors are fixed automatically. 
 
-VSCode extension: [TSLint on visual studio marketplace](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin).
-Enabling "fix on save" in VSCode: [auto fix and auto fix on save](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin#auto-fix-and-auto-fix-on-save).
+**Useful resources:**
+- [TSLint extension for VSCode](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin).
+- [Enabling "fix on save" in VSCode](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin#auto-fix-and-auto-fix-on-save).
 
-**VSCode "workspace library execution"-issue:** 
+#### VSCode "workspace library execution"-issue
 An issue might occur where TSLint will randomly disable for the current workspace and has to be enabled again using the command `TSLint: Manage workspace library execution`. 
 
 To solve this: 
 1. open a typescript file in VScode. 
-2. open the command pallette (<kbd>ctrl/cmd</kbd> + <kbd>shift</kbd> + <kbd>p</kbd>) and enter: `typescript: select typescript version`.
-3. select: `use workspace version 3.6.5`.
-4. open the command pallette (<kbd>ctrl/cmd</kbd> + <kbd>shift</kbd> + <kbd>p</kbd>) and enter: `TSLint: Manage workspace library execution`.
-5. select: `always enable workspace library execution`.
+2. open the command pallette (<kbd>ctrl/cmd</kbd> + <kbd>shift</kbd> + <kbd>p</kbd>) and enter: <br />
+`typescript: select typescript version`.
+4. select: `use workspace version 3.6.5`.
```

Comment:
Once the Angular (and Typescript) upgrade issue is merged I don't think this version number is correct any more.

Reply:
I will add a repeating todo to my todo-system, to check up on the angular upgrade issue, and update this if necessary :D